src/atcore.cpp

```diff
@@ -324,3 +324,31 @@ void AtCore::detectFirmware()
     connect(serial(), &SerialLayer::receivedCommand, this, &AtCore::findFirmware);
     requestFirmware();
 }
+/*~~~~~Control Slots ~~~~~~~~*/
+void AtCore::home()
+{
+    pushCommand(GCode::toCommand(GCode::G28));
+}
+
+void AtCore::home(QString axis)
+{
+    axis.replace(QChar::fromLatin1('X'), QStringLiteral("X0"));
+    axis.replace(QChar::fromLatin1('Y'), QStringLiteral("Y0"));
+    axis.replace(QChar::fromLatin1('Z'), QStringLiteral("Z0"));
+    pushCommand(GCode::toCommand(GCode::G28, axis));
+}
```

Comment:
Using a string isn't the best way to perform that, maybe:
```
printer{
	axis{
   		uchar X = 1<<0;
   		uchar Y = 1<<1;
   		uchar Z = 1<<2;
   		uchar E = 1<<3;
	}
}
```
And use the home or move function like:
```
move(printer::X, 12);
move(printer::X | printer::Z, 12);
home(printer::X | printer::Z); // Will not work with E
```